.github/workflows/build.yml

```diff
@@ -22,49 +29,62 @@ jobs:
         with:
           name: version-txt
           path: version.txt
-  code-check-rust:
-    runs-on: "ubuntu-latest"
-    steps:
-      - uses: actions/checkout@v2
-      - name: Check format
-        run: |
-          rustup toolchain install nightly --component rustfmt
-          rustup default nightly
-          cargo fmt -- --check
-  build-rust:
-    needs: ["materialize-build-version", "code-check-rust"]
+  build:
+    needs: ['codestyle-check', 'version']
     runs-on: ${{ matrix.os }}
-    continue-on-error: true
```

Comment:
continue-on-error is exactly the wrong semantics; we want the build to fail if there's a failure in any of the matrix, but we don't want it to fail-fast: let the rest of the matrix continue!